app/services/analytics_report_service.rb

```diff
@@ -16,26 +16,12 @@ def analytics_report_json
     {
       start_date: report_since.to_date,
       end_date: Date.today,
-      ctas: cta_json,
       visits: visits_json,
       users: users_json,
       macros: macros_json
     }
   end
 
-  def cta_json
```

Comment:
@Pralish can you spec out/estimate how we can add analytics to API namespaces/resources? This is not a top priority but I'd like to have it done before the Violet Rails 1.0 release